### PR TITLE
ENGDOCS-2181c

### PR DESCRIPTION
**Pull request by aevesdocker**
## Description

Replaces https://github.com/docker/docs/pull/20646 that was so old it was still using the `toc.yaml` file. 
SME reviews complete. Just need editorial 

## Reviews

- [ ] Technical review
- [x] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    badge:
+      color: violet
+      text: EA
+---
+
+{{% restricted title="Early Access" %}}
+Settings Management in the Docker Admin Console is an [early access](/release-lifecycle#early-access-ea) feature and is available to Docker Business customers only.
+{{% /restricted %}}
+
+This page contains information for administrators on how to configure Settings Management with the Docker Admin Console to specify and lock configuration parameters to create a standardized Docker Desktop environment across your Docker company or organization.
```
Comment (sarahsanders-docker) [inline content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md:18]:
```suggestion
This page contains information for administrators on how to configure Settings Management with the Docker Admin Console. You can specify and lock configuration parameters to create a standardized Docker Desktop environment across your company or organization.
```

Comment (sarahsanders-docker) [reply content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md:18]: just a suggestion for readability

File: content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Prerequisites
+
+- [Download and install Docker Desktop 4.36.0 or later](/manuals/desktop/release-notes.md).
+- Make sure you have [verified your domain](/manuals/security/for-admins/single-sign-on/configure.md#step-one-add-and-verify-your-domain)
```
[the remark this answers is not in the document]
Comment (sarahsanders-docker) [reply content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md:23]: just a suggestion for simplicity

File: content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- [Enforce sign-in](/manuals/security/for-admins/enforce-sign-in/_index.md). This is
+because the Settings Management feature requires a Docker Business
+subscription and therefore your Docker Desktop users must authenticate to your
+organization for configurations to take effect.
```
Comment (sarahsanders-docker) [inline content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md:27]:
```suggestion
- [Enforce sign-in](/manuals/security/for-admins/enforce-sign-in/_index.md). The Settings Management feature requires a Docker Business subscription, therefore your Docker Desktop users must authenticate to your
organization for configurations to take effect. 
```

File: content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+   > [!NOTE]
+   >
+   > If a settings policy is assigned to all users, it sets the policy as the global default policy. You can only have one global settings policy at a time. 
+   > If a user already has a user-specific settings policy assigned, the user-specific policy takes precedence over a global policy. 
+
+   > [!TIP]
+   >
+   > Before setting a global settings policy, it is recommended that you first test it as a user-specific policy to make sure you're happy with the changes before proceeding. 
+
+6. Configure the settings for the policy. Go through each setting and select your chosen setting state. You can choose
```
Comment (sarahsanders-docker) [inline content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md:53]:
```suggestion
6. Configure the settings for the policy. Go through each setting and select your chosen setting state. You can choose:
```

File: content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  > [!IMPORTANT]
+  >
+  > Selecting **Restart** from the Docker menu isn't enough as it only restarts some components of Docker Desktop.
+
+To avoid disrupting your users' workflows, Docker doesn't automatically require that users re-launch once a change has been made.
+
+> [!NOTE]
+>
+> Settings are synced to Docker Desktop and the CLI when a user is signed in and starts Docker Desktop, and then every 60 minutes. 
+If your settings policy needs to be rolled back, either delete the policy or edit the policy to set individual settings to **User-defined**.
```
Comment (sarahsanders-docker) [inline content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md:74]:
```suggestion

If your settings policy needs to be rolled back, either delete the policy or edit the policy to set individual settings to **User-defined**.
```

Comment (sarahsanders-docker) [reply content/manuals/security/for-admins/hardened-desktop/settings-management/configure-admin-console.md:74]: whitespace nit

**Review by sarahsanders-docker — CHANGES_REQUESTED** [review]
looks good! left a few nits but overall the information is clear and easy to follow :)

File: _vale/config/vocabularies/Docker/accept.txt [hunk 6]
```diff
@@ -24,6 +24,7 @@ Ddosify
 Debootstrap
 Dev Environments?
 Django
+Docker Admin Console
```
Comment (dvdksn) [inline _vale/config/vocabularies/Docker/accept.txt:27]: Not needed; fixed in [#21422](https://github.com/docker/docs/pull/21422)

**Review by sarahsanders-docker — APPROVED** [review]
LGTM! :)

**Pull request merged**